packages/medusa/src/api-v2/admin/price-lists/route.ts

```diff
@@ -27,3 +30,32 @@ export const GET = async (
     limit,
   })
 }
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminPostPriceListsReq>,
+  res: MedusaResponse
+) => {
+  const workflow = createPriceListsWorkflow(req.scope)
+  const priceListsData = [req.validatedBody]
+
+  const { result, errors } = await workflow.run({
+    input: { priceListsData },
+    throwOnError: false,
+  })
+
+  if (Array.isArray(errors) && errors[0]) {
+    throw errors[0].error
+  }
+
+  const [[priceList]] = await listPriceLists({
```

Comment:
suggestion(non-blocking): Use `remoteQuery`

Reply:
This function is already calling remote query inside

Reply:
Ah my bad. Discard this and the other comments then.